default.json

```
@@ -2,7 +2,7 @@
   "labels": ["maintenance"],
   "extends": ["config:base", ":disableDependencyDashboard"],
   "lockFileMaintenance": {
-    "enabled": false
+    "enabled": true
```

this will create lots of noise. Maybe we can limit it to once per week, as recommended here: https://docs.renovatebot.com/configuration-options/#lockfilemaintenance
```suggestion
    "enabled": true,
    "schedule": ["before 5am on monday"],
```